doc/content/getting-started/troubleshooting/_index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Troubleshooting"
+description: ""
+---
+
+This section provides help for common issues and frequently asked questions you may have when getting started with {{% tts %}}. 
+
+<!--more-->
+
+If you encounter a bug, please [file an issue](https://github.com/TheThingsNetwork/lorawan-stack/issues/new/choose). If you have questions not covered here, please ask in [{{% tts %}} Forum](https://www.thethingsnetwork.org/forum/c/network-and-routing/v3). To check the status of cloud services, see our [status page](https://status.thethings.industries/).
+
+## Troubleshooting {{% tts %}}
+
+### How do I generate and authorize OAuth client keys?
+
+See the [authentication reference]({{< ref src="reference/api/authentication" >}}).
+
+### How do I create and authorize new users?
+
+See [user and organization management]({{< ref src="getting-started/user-management" >}}).
+
+## Troubleshooting Devices
+
+### How do I see device events?
+
+Device logs can be found in the console in the device's general information page.
+
+To view gateway logs with the CLI, use the following command:
+
+```bash
+$ ttn-lw-cli events subscribe --application-id <application ID> --device-id <device ID>
+```
```

Comment:
We have Console for this right? Also isn't this documented so that we can refer to it?